index_test.go

```diff
@@ -402,7 +402,7 @@ func TestAndSearch(t *testing.T) {
 			ContentBytesLoaded: 18,
 			IndexBytesLoaded:   8,
 			NgramMatches:       3, // we look at doc 1, because it's max(0,1) due to AND
-			MatchCount:         1,
+			MatchCount:         2,
```

Comment:
Note: this is the only test that changed by generating line matches from chunk matches, and that's because we count ranges (the equivalent of fragments) rather than matched files. I can change this, but it seems like the more consistent thing to do.